### PR TITLE
initial upload of dcam.html

### PR DESCRIPTION
**Pull request by Danny2097**
additional page containing D-Campbell's technical skills

### CODE REVIEW

**Comment by walshd** [comment]
Cool thanks Dan

**Comment by walshd** [comment]
Hi Dan, after review, could you please just check out the spelling mistake you made.

File: dcam.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en"><head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>DWal</title>
```
Comment (Danny2097) [inline dcam.html:6]: Title tag is incorrect

File: dcam.html [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    <li>css</li>
+    <li>js</li>
+    <li>php</li>
+    <li>python</li>
+    <li>Data Science</li>
+    <li>Machine Learning</li>
+    <li>Artifical Intelligence</li>
+    <li>Software Engineering</li>
+  </ol>
+
```
Comment (Danny2097) [inline dcam.html:21]: Remove whitespace

**Review by Danny2097 — COMMENTED** [review]
Checkout some of the issues highlighted

**Pull request merged**